### DISCUSSION TITLE
Copilot doesn't give Suggestions

### DISCUSSION THREAD

**Discussion by manuthebyte** [community/community#14400] [Copilot Conversations] [answered]
Hello there!
I am running into a problem at the moment with Github Copilot. Everytime I wait for a suggestion it says "Github Copilot has a temporary problem".

And when I open the output of GitHub Copilot it throws this error:
`[ERROR] [fetch] [2022-04-06T14:24:41.347Z] Unhandled status from server:,429,{"error":{"message":"Requests to the Create a completion from a chosen model Operation under OpenAI Language Model Instance API have exceeded rate limit of your current OpenAI S0 pricing tier. Please retry after 1 second. Please contact Azure support service if you would like to further increase the default rate limit."}}`

**Comment by davecheney** [comment, marked as answer]
Duplicate of https://github.com/github/feedback/discussions/14370
